test/integration/performance/tcp/tcp.go

```diff
@@ -0,0 +1,165 @@
+package tcp
+
+import (
+	"context"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/skupperproject/skupper/api/types"
+	"github.com/skupperproject/skupper/test/utils"
+	"github.com/skupperproject/skupper/test/utils/base"
+	"gotest.tools/assert"
+)
+
+const (
+	ENV_IPERF_PARALLEL_CLIENTS = "IPERF_PARALLEL_CLIENTS"
+	ENV_IPERF_TRANSMIT_SIZES   = "IPERF_TRANSMIT_SIZES"
+	ENV_IPERF_WINDOW_SIZE      = "IPERF_WINDOW_SIZE"
+	ENV_IPERF_MEMORY           = "IPERF_MEMORY"
+	ENV_IPERF_CPU              = "IPERF_CPU"
+	ENV_IPERF_JOB_TIMEOUT      = "IPERF_JOB_TIMEOUT"
+
+	ENV_SKUPPER_SITES              = "SKUPPER_SITES"
+	ENV_SKUPPER_MAX_FRAME_SIZE     = "SKUPPER_MAX_FRAME_SIZE"
+	ENV_SKUPPER_MAX_SESSION_FRAMES = "SKUPPER_MAX_SESSION_FRAMES"
+	ENV_SKUPPER_MEMORY             = "SKUPPER_MEMORY"
+	ENV_SKUPPER_CPU                = "SKUPPER_CPU"
+)
+
+func TestMain(m *testing.M) {
+	log.Println("BATATA")
```

Comment:
Banana works better here !!

Reply:
I knew I should have used a fruit instead. :)